docs/lib/restapi/fragments/android/fetch.md

```diff
@@ -0,0 +1,62 @@
+## GET requests
+
```

Comment:
I know this is obvious, but all code samples should have some descrition about what it is doing.  For example, we could add the following:

To make a GET request, first create a `RESTRequest` object and then use the `Amplify.API.get` api to issue the request: